### PR TITLE
Implement depth bias control

### PR DESCRIPTION
**Pull request by Joshua-Ashton**
Marked as draft as have not tested yet, but should be there.

### CODE REVIEW

File: src/d3d9/d3d9_util.h [hunk 1]
```diff
@@ -176,14 +176,14 @@ namespace dxvk {
...
       case VK_FORMAT_D16_UNORM:
-        return vendorId == 0x10de ? float(1 << 15) : float(1 << 16);
+        return (vendorId == 0x10de && !exact) ? float(1 << 15) : float(1 << 16);
 
       case VK_FORMAT_D24_UNORM_S8_UINT:
-        return vendorId == 0x10de ? float(1 << 23) : float(1 << 24);
+        return (vendorId == 0x10de && !exact) ? float(1 << 23) : float(1 << 24);
 
       default:
       case VK_FORMAT_D32_SFLOAT_S8_UINT:
```
Comment (DadSchoorse) [inline src/d3d9/d3d9_util.h:189]: D32f with FORCE_UNORM uses 1 << 24.

Comment (Joshua-Ashton) [reply src/d3d9/d3d9_util.h:189]: Fixed,

**Review by K0bin — APPROVED** [review]
Code looks good to me. I'm still waiting for Nvidia to release a new beta driver to actually test it though.

**Comment by Blisto91** [comment]
Verified that the 3 AMD relevant ones i could test are fixed
https://github.com/doitsujin/dxvk/issues/2642
https://github.com/doitsujin/dxvk/issues/1560
https://github.com/doitsujin/dxvk/issues/2941

I have never been able to get Counter-Strike Online 2 working on Linux so https://github.com/doitsujin/dxvk/issues/1940 will have to wait for confirmation until original reporter can test with the Windows AMD driver. But it is expected that it will be fixed.

Nvidia testing is pending driver support :frog:

**Comment by Joshua-Ashton** [comment]
https://github.com/doitsujin/dxvk/issues/1461

It is marked as NV only but knowing it still works on AMD is good.

**Comment by Blisto91** [comment]
Looks good on AMD. Didn't notice any issues running around for a bit.

<details>
  <summary>Screenshot</summary>

image

</details>

**Review by doitsujin — APPROVED** [review]

**Comment by mrdeathjr28** [comment]
In my case appear enabled with anv

dpcen

**Pull request merged**